### ISSUE TITLE
Version 3.3.0 sub-window min/max/close buttons not rendering correctly

### ISSUE THREAD

**Issue by tedstriker2** [pawelsalawa/sqlitestudio#3995]
### Details
When a sub-window is maximized, the minimize, maximize, and close buttons are not rendered correctly.

### Steps to reproduce
Maximize a window. Here's a screenshot:
![image](https://user-images.githubusercontent.com/56124863/109013806-2a6f7500-7679-11eb-8aab-9ed50ee6aca6.png)

### Operating system
Windows 10

### SQLiteStudio version
3.3.0

**Comment by pawelsalawa** [comment]
1. What screen resolution do you have?
2. Do you have Windows interface autoscaling enabled? I mean this: ![image](https://user-images.githubusercontent.com/481846/109068716-86291500-76f0-11eb-9086-1bcf61da4537.png)
3. Does this also happen if you switch the style to Fusion? (hit F2 -> go to Look & Feel -> Styles -> style dropdown)

**Comment by tedstriker2** [comment]
1. Screen resolution 1920x1080
2. Windows scaling 125%
3. It does not happen wit the "Fusion" style. The issue is with the style named "windowsvista".

**Comment by pawelsalawa** [comment]
@tedstriker2 Thanks. Could you pleae check just one last thing? If you turn the scaling down to 100%, does this still happen with "windowsvista"?

**Comment by tedstriker2** [comment]
> 
> 
> @tedstriker2 Thanks. Could you pleae check just one last thing? If you turn the scaling down to 100%, does this still happen with "windowsvista"?

With scaling down from to 100%, "windowsvista" button are rendered correctly.

If it helps, the buttons were rendered correctly at 125% scaling in version 3.0.6.

**Comment by pawelsalawa** [comment]
Yeah, I have an idea why it might be. Thanks for the help. Fix scheduled for 3.3.1.

**Comment by pawelsalawa** [comment]
The problem is in Qt and it's not the only problem with HighDPI that Qt has currently (https://bugreports.qt.io/browse/QTBUG-80228). Hopefully the situation will improve as new Qt versions are released.

In the meanwhile, the very problem described in this issue is actually easy to work around it. You just need to restart the SQLiteStudio. The problem manifests itself only if you run SQLiteStudio without any MDI windows and you open new window (while working in HighDPI display). If you run SQLiteStudio with already open MDI windows (from previous run), the buttons will be rendered correctly (most of the time...).

Now we can only wait for fix from Qt.

**Comment by pawelsalawa** [comment]
The Qt issue mentioned above seems to be resolved in Qt 6.4.

**Comment by pawelsalawa** [comment]
As master branch (3.5.0) is already using Qt 6 I can confirm it's fixed.
